src/web/screens/sharingPermissions.tsx

```diff
@@ -27,6 +30,10 @@ function SharingPermissions() {
   const { participant } = useContext(ParticipantContext);
   const [sharingParticipants, setSharingParticipants] = useState<AvailableParticipantDTO[]>([]);
   const [statusPopup, setStatusPopup] = useState<StatusPopupType>();
+  const { participants, participantTypes } = useLoaderData() as {
+    participants: AvailableParticipantDTO[];
+    participantTypes: ParticipantType[];
```

Comment:
maybe we can use ParticipantTypeDTO instead of ParticipantType here?

Reply:
done